core/services/functions/listener_test.go

```diff
@@ -46,18 +47,22 @@ type FunctionsListenerUniverse struct {
 	pluginORM      *functions_mocks.ORM
 	logBroadcaster *log_mocks.Broadcaster
 	ingressClient  *sync_mocks.TelemetryIngressClient
+	decryptor      *threshold_mocks.Decryptor
 }
 
 func ptr[T any](t T) *T { return &t }
 
 var (
-	RequestID         functions_service.RequestID = newRequestID()
-	RequestIDStr      string                      = fmt.Sprintf("0x%x", [32]byte(RequestID))
-	SubscriptionOwner common.Address              = common.BigToAddress(big.NewInt(42069))
-	SubscriptionID    uint64                      = 5
-	ResultBytes       []byte                      = []byte{0xab, 0xcd}
-	ErrorBytes        []byte                      = []byte{0xff, 0x11}
-	Domains           []string                    = []string{"github.com", "google.com"}
+	RequestID            functions_service.RequestID = newRequestID()
+	RequestIDStr         string                      = fmt.Sprintf("0x%x", [32]byte(RequestID))
+	SubscriptionOwner    common.Address              = common.BigToAddress(big.NewInt(42069))
+	SubscriptionID       uint64                      = 5
+	ResultBytes          []byte                      = []byte{0xab, 0xcd}
+	ErrorBytes           []byte                      = []byte{0xff, 0x11}
+	Domains              []string                    = []string{"github.com", "google.com"}
+	EncryptedSecretsUrls []byte                      = []byte{0x5f, 0x91, 0x82, 0xcb, 0xe8, 0x31, 0x34, 0xb3, 0x2f, 0xad, 0x55, 0x9d, 0xa4, 0x40, 0x8e, 0x8f, 0x02, 0xca, 0xae, 0x7c, 0xdd, 0x1f, 0x60, 0x21, 0x55, 0xab, 0x02, 0x1d, 0x97, 0x41, 0xbd, 0x3b, 0x47, 0xf9, 0xe7, 0x5b, 0x86, 0xd7, 0x08, 0x0b, 0xbe, 0xcf, 0xed, 0xbd, 0xaf, 0x25, 0x58, 0x97, 0x60, 0xfc, 0x03, 0x48, 0x62, 0xed, 0x46, 0x34, 0x4b, 0x05, 0x97, 0xd6, 0x2c, 0x10, 0xc3, 0x42, 0x0a, 0xfa, 0xb4, 0x7b, 0x1f, 0x2e, 0xd4, 0xd7, 0x11, 0x51, 0x34, 0xb1, 0xa3, 0xae, 0xfc, 0x97, 0x7c, 0x73, 0x36, 0x38, 0xef, 0xd6, 0x65, 0xb8, 0x2c, 0x3f, 0x19, 0xfb, 0xb0, 0x5e, 0x36, 0x5f, 0x25, 0x1a, 0x5b, 0x1e, 0xe1, 0x3b, 0x21, 0x5d, 0xe5, 0x6d, 0x7a, 0xd9, 0x97, 0xbe, 0xcb, 0x03, 0xef, 0x5e, 0x49, 0x00, 0x87, 0x92, 0xdd, 0xe1, 0x23, 0x3c, 0x7a, 0x3a, 0xf0, 0x98, 0xce, 0xcc, 0x10, 0x9b, 0x4b, 0x49, 0x5f, 0xf9, 0x2e, 0xd3, 0xc8, 0xca, 0x11, 0xd0, 0x13, 0x8e}
```

Comment:
You don't need those huge real values here since you're mocking everything. Just create fake ones that are much smaller.